polartools/diffraction.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2020, UChicago Argonne, LLC.
+# See LICENSE file for details.
+
+import numpy as np
+from pandas import DataFrame
+from lmfit.models import GaussianModel, LorentzianModel, VoigtModel, LinearModel, PseudoVoigtModel
+import matplotlib.pyplot as plt
+from .load_data import load_table
+
+def fit_peak(xdata,ydata,model='Gaussian',output=False):
```

Comment:
It'd be useful to pass the plot axis somewhere. One option is that the user creates the figure and gives it to `fit_peak`. `output` would take a figure instance the user would run something like:
```
figure, axis = plt.subplots(1,1)
results = fit_peak(xdata, ydata, model='Lorentzian', output=axis)
```
Or the figure could be created inside `fit_peak` and returned to the user:
```
results, figure, axis = fit_peak(xdata, ydata, model='Lorentzian', output=True)
```
I sort of prefer the first option because that way the number of returned parameters is the same whether you plot the results or not.

Reply:
This should just be for check while fitting, if fit is ok.